src/rest/connector/libs/nd/implementation.py

```diff
@@ -45,13 +45,14 @@ class Implementation(Imp):
     """
 
     @BaseConnection.locked
-    def connect(self, timeout=30, retries=3, retry_wait=10):
+    def connect(self, timeout=30, retries=3, retry_wait=10, verify=False):
         """connect to the device via REST
         Arguments
         ---------
             timeout (int): Timeout value
             retries (int): Max retries on request exception (default: 3)
             retry_wait (int): Seconds to wait before retry (default: 10)
+            verify (bool): defaults to False
```

Comment:
Defaults to False, but what does it do? A brief description of what we're verifying would be ideal